### DISCUSSION TITLE
Portuguese (Portugal) sentences that should be supported

### DISCUSSION THREAD

**Discussion by marcocunha** [OHF-Voice/intents#391] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your Portuguese (pt-pt) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by joaorgoncalves** [comment]
@marcocunha let's open this thread to all PT variations and not only pt-pt?

**Reply by marcocunha** [reply]
@joaorgoncalves there are some significant differences for other variations like pt-br. We should keep them separate and even have native speakers contributing to these ones.
